.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,24 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
```

Comment:
I'd add a little more information, such as:
- **Describe the bug** - A clear and concise description of what the bug is.
- **To Reproduce** - Provide a code snippet that reproduces the issue.
- **Expected behavior** - A clear and concise description of what you expected to happen.
- **Screenshots** - If applicable, add screenshots to help explain your problem.
- **Environment Info** - list of installed libraries with their corresponding versions, OS info and stuff (maybe create a script for checking user env and write to use the output from it here).
- **Additional context** - add any other context about the problem here.